protos/openoffload.proto

```diff
@@ -146,7 +146,8 @@ message statisticsRequestArgs{
   //  pageSize = 0 will turn off paging
   //  does paging make sense for a stream ?
   //  the client should read/process each event on the stream anyway.
-  int32  pageSize=1;
+  uint32  pageSize=100;
+  uint32  page=1;
```

Comment:
do does page=0 turn it off ? is that in the README.md ?

Reply:
The user would continue to get streams of sessions by increasing the page number until they are returned < pageSize.  At that point they know there are no more sessions.

Reply:
Yes. The user would continue to get streams of sessions by increasing the page number until they are returned < pageSize. At that point they know there are no more sessions.